tests/components/notify/test_init.py

```diff
@@ -107,9 +299,14 @@ async def test_remove_targets(hass: HomeAssistant) -> None:
 
 
 class NotificationService(notify.BaseNotificationService):
-    """A test class for notification services."""
-
-    def __init__(self, hass, target_list={"a": 1, "b": 2}, name="notify"):
+    """A test class for legacy notification services."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        target_list=copy.copy({"a": 1, "b": 2}),
```

Comment:
We shouldn't put a mutable object in a default argument. Change this to default to `None` instead and set the `target_list` to `{"a": 1, "b": 2}` if it's `None` in the init method.

Reply:
You are right, i'll correct it

Reply:
In fact I only changed the typing of the existing code. I could have addressed this in a separate PR.